.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Feature request
+about: Suggest a new feature for this project
+title: "[Feature Request]"
+labels: feature
+assignees: ''
```

Comment:
Provide a list of core developers and their area of expertise to help contributors choose between them in case of urgent feedback

Reply:
Done